CMB2/CMB2
# Update shiftRows functionality to be more simple, and fix issues with JS initialization. Fixes #1426

## Description
Fixes issues with repeatable and sortable group with repeatable fields while shifting group. Fixes #1426

## Testing procedure
1. Add a bunch of field types (preferably all of them?) to the a group field.
	1. Make sure some of them are set to be repeatable.
	1. e.g.:
	```php
	add_action( 'cmb2_admin_init', function() {

		$cmb = new_cmb2_box([
			'id'            => 'metabox_attachment',
			'title'         => 'Poo poo',
			'object_types'  => [ 'page' ], // Post type
			'priority'      => 'high',
			'show_names'    => true
		] );

		$group_field_id = $cmb->add_field( array(
			'before_group' => function( $args, $field ) {
				echo '<xmp>'. __LINE__ .') $field->value: '. print_r( $field->value, true ) .'</xmp>';
			},
			'id'          => 'wiki_test_repeat_group_1429',
			'type'        => 'group',
			'description' => __( 'Generates reusable form entries', 'cmb2' ),
			'repeatable'  => true,
			'options'     => array(
				'group_title'   => __( 'Entry {#}', 'cmb2' ),
				'add_button'    => __( 'Add Another Entry', 'cmb2' ),
				'remove_button' => __( 'Remove Entry', 'cmb2' ),
				'sortable'      => true,
			),
		) );

		$cmb->add_group_field( $group_field_id, array(
				'name' => 'Entry Title',
				'id'   => 'subtitle',
				'type' => 'text',
				'repeatable' => true,
		) );

		$cmb->add_group_field( $group_field_id, array(
			'name'       => 'Test Image',
			'desc'       => 'Upload an image or enter a URL.',
			'id'         => 'demo_image',
			'type'       => 'file',
			'preview_size'    => array( 50, 50 ),
		) );
		$cmb->add_group_field( $group_field_id, array(
			'name'         => 'Multiple Files',
			'desc'         => 'Upload or add multiple images/attachments.',
			'id'           => 'demo_file_list',
			'type'         => 'file_list',
		) );

		$cmb->add_group_field( $group_field_id, array(
			'name'    => 'Color',
			'id'      => 'demo_color',
			'type'    => 'colorpicker',
			'default' => '#ffffff',
		) );
	});
	```

1. In first group click to repeat field and populate both https://prnt.sc/13o8bn2
1. Add another group ( Add another Entry )
1. Click move down on first group.
1. See that all works as expected.
1. After moving fields around, try click/dragging repeatable field items
1. Once tested for a bit, save the post.
1. See that data is saved properly.


## Types of changes
- **Bug fix (non-breaking change which fixes an issue)**

## Checklist:
